nncf/openvino/graph/model_transformer.py

```diff
@@ -592,6 +594,56 @@ def _apply_model_extraction_transformation(
         copy_rt_info(model, extracted_model, path=["nncf"])
         return extracted_model
 
+    @staticmethod
+    def _apply_model_extraction_transformation_v2(
```

Comment:
Yes, The first version is needed and used in the bias correction algorithm. I hope we rework this place during migration the bias correction algorithm to layer-wise engine.

The `_apply_model_extraction_transformation_v2` processes the `OVModelExtractionCommandV2` command, which is specified by input nodes and their **output ports** instead of the input nodes and their **input ports** in the `OVModelExtractionCommand` command.

I have added this comment to the scope of work on the task of migrating the bias correction algorithm to the layer-wise engine.